docs/DeveloperGuide.md

```diff
@@ -312,10 +329,28 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 * 3a. The given index is invalid.
 
-    * 3a1. AddressBook shows an error message.
+    * 3a1. NUSContacts shows an error message.
 
       Use case resumes at step 2.
 
+**UC3 Find a tutor by name**
+
+**MSS**
+
+1.  User requests to find a tutor
+2.  NUSContacts prompts search criterion (name or course code)
+3.  User provides the search criterion
+4.  NUSContacts shows the tutor(s) that match the criterion
+
+    Use case ends.
+
+**Extensions**
+
+* 4a. No tutor matches the criterion.
+    * 4a1. NUSContacts informs the student of the error.
```

Comment:
Since we're using "user" throughout, it's better to stick to it instead of "student".

Reply:
Changed all instances of `student`  back to `user`